scalameta/contrib/src/main/scala/scala/meta/contrib/ScaladocParser.scala

```diff
@@ -0,0 +1,120 @@
+package scala.meta.contrib
+
+import DocToken._
+import scala.meta.Syntax
+import scala.meta.tokens.Token.Comment
+
+object ScaladocParser {
+
+  /**
+    * Parses a scaladoc [[Comment]].
+    */
+  def parseScaladoc(comment: Comment): Seq[DocToken] = {
+
+    val trimmedComment: String = comment.show[Syntax].trim
+
+    require(
+      requirement = trimmedComment.startsWith("/*") && trimmedComment.endsWith("*/"),
+      message = "Input comment is not a Scaladoc"
+    )
+
+    val scaladocSymbols = Seq('/', '*', ' ')
+
+    val lineSeparedDocString: Seq[String] =
+      trimmedComment
+        .split(Array('\n', '\r')) // Splits the Scaladoc in lines
```

Comment:
It works :)

The only problem is that I still not understand what this is exactly doing: 
`((AnyChar~ !"}}}").rep ~ AnyChar)`

Thank you a lot, with this I will be able to continue working with the PR :)